doc/examples/effects/infinite_effects/lib/square.dart

```diff
@@ -6,16 +6,20 @@ import 'dart:ui';
 class Square extends PositionComponent {
   final Paint _paint;
 
-  Square(this._paint, double x, double y) {
+  Square(this._paint, double x, double y, {double angle = 0.0}) {
     width = 100;
     height = 100;
     this.x = x;
     this.y = y;
+    this.angle = angle;
     anchor = Anchor.center;
   }
 
   @override
   void render(Canvas canvas) {
+    canvas.translate(x, y);
```

Comment:
This should not be needed, `PositionComponent` already has an angle that is already makes the component rotate.

Reply:
you need to call prepareCanvas to use that (and we should!)

in v1.0 it will be super.render :p (spoilers)

Reply:
Hmm, that is true, somehow I thought PositionComponent already did that.

I wonder if shouldn't we add a render method on PositionComponent, with this call and a @mustCallSuper already on the 0.22.0 version as we don't know when 1.0 will land, and this is not a very big breaking change.

Thoughts @luanpotter ?

Reply:
that is a VERY big breaking change! it will break a lot of games. that is why I packaged it onto v1.0.

Reply:
It will break ALL the games, but IMO, it is not common for people to use PositionComponent directly, and even people who uses it, will have to just add this line to fix: `super.render(canvas)`, so it is a very easy fix. That is why I consider this very problematic to add prior to v1, as it would be an easy fix, and probably not many people would be affected. But I am fine with not adding this prior to v1...